test/integration/targets/git/vars/main.yml

```diff
@@ -15,10 +15,10 @@ separate_git_dir: '{{ output_dir }}/sep_git_dir'
 repo_format1: 'https://github.com/jimi-c/test_role'
 repo_format2: 'git@github.com:jimi-c/test_role.git'
 repo_format3: 'ssh://git@github.com/jimi-c/test_role.git'
-repo_submodules: 'https://github.com/abadger/test_submodules.git'
```

Comment:
@abadger we probably need to put test repos under our org or create a separate one for those.